### ISSUE TITLE
Lab 14 when starting Native Image warning: [main] WARN GC notifications because MemoryPoolMXBeans are not provided by the JVM

### ISSUE THREAD

**Issue by amitpal-source** [graemerocher/micronaut-oraclecloud-workshop#39]
When starting Native Image there is a GC warning 
[main] WARN  i.m.c.i.binder.jvm.JvmGcMetrics - GC notifications will not be available because MemoryPoolMXBeans are not provided by the JVM

this is only seen when ./build/native-image/application
not for java -jar build/libs/demo-0.1-all.jar

The native image starts up and I can access the URL no issues there...

**Comment by graemerocher** [comment]
yeah graal doesn't support JMX yet, not sure there is much we can do about this
